objcgen/objcgenerator-processor.cs

```diff
@@ -230,9 +230,7 @@ protected IEnumerable<FieldInfo> GetFields (Type t)
 			}
 		}
 
-		List<ProcessedType> enums = new List<ProcessedType> ();
```

Comment:
glad to hear, we need to discuss the next steps to split the processor from the generator (and finally get rid of potential dupes)